src/controller/ExampleOperationalCredentialsIssuer.h

```diff
@@ -56,8 +56,8 @@ class DLL_EXPORT ExampleOperationalCredentialsIssuer : public OperationalCredent
     ~ExampleOperationalCredentialsIssuer() override {}
 
     CHIP_ERROR GenerateNOCChain(const ByteSpan & csrElements, const ByteSpan & csrNonce, const ByteSpan & attestationSignature,
-                                const ByteSpan & attestationChallenge, const ByteSpan & DAC, const ByteSpan & PAI,
-                                Callback::Callback<OnNOCChainGeneration> * onCompletion) override;
+                                const ByteSpan & attestationChallenge, const ByteSpan & attestationElements, const ByteSpan & DAC,
```

Comment:
You cannot do both at the same time with this API: Both have a different signature. You cannot simply pass attestationElements without passing the attestation signature that came in attestation response. Otherwise you are using 1 signature over a different payload (the CSR elements) to trust the attestation elements. You would need both signatures to be passed.